### PR TITLE
feature/news: Implement new design of complete form for redactor and space manager

### PR DESCRIPTION
**Pull request by aelbenney**

### CODE REVIEW

File: webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue [hunk 1]
```diff
@@ -150,36 +147,46 @@ export default {
...
+        height: this.newsFormSummaryHeight ,
+        sharedSpaces: {
+          top: 'newsTop'
+        },
+        on: {
+          change: function (evt) {
+            self.newsActivity.summary = evt.editor.getData();
+          }
+        }
+      });
```
Comment (thomasdelhomenie) [inline webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue]: We should factorize the ckeditor configuration.

File: webapp/src/main/webapp/news-activity-composer-app/components/customConfig/customPlugin/sharedSSpace/plugin.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+					a.on("loaded", function() {
+						var b = a.config.sharedSpaces;
+						if (b)
+							for (var c in b)
+								f(a, c, b[c])
+					}, null, null, 9)
+				}
+			})
+		}
+)();
```
Comment (thomasdelhomenie) [inline webapp/src/main/webapp/news-activity-composer-app/components/customConfig/customPlugin/sharedSSpace/plugin.js]: This file should be removed.

Comment (aelbenney) [reply webapp/src/main/webapp/news-activity-composer-app/components/customConfig/customPlugin/sharedSSpace/plugin.js]: okay!

File: webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue [hunk 3]
```diff
@@ -1,70 +1,64 @@
...
-            <div class="newsFormActions">
-              <a id="newsPlus" :data-original-title="extendFormButtonTooltip" class="btn btn-primary"
-                 rel="tooltip" data-placement="bottom"
-                 @click="extendedForm = !extendedForm;">
-                <i :class="extendFormButtonClass"></i>
-              </a>
-              <button id="newsPost" :disabled="postDisabled" class="btn btn-primary"> {{ $t("news.composer.post") }}
-              </button>
+      </div>
+      <div id="newsTop"></div>
```
[the remark this answers is not in the document]
Comment (aelbenney) [reply webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue]: I put it with the fixed top bar! to make it easy to fix all the block when scrolling

[the remark this answers is not in the document]
Comment (thomasdelhomenie) [reply webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue]: Ah yes ok :+1:

File: webapp/src/main/webapp/css/news.less [hunk 4]
```diff
@@ -251,7 +252,6 @@
   display: flex;
   flex-direction: column-reverse;
   position: relative;
-  width: 100%;
```
Comment (aelbenney) [inline webapp/src/main/webapp/css/news.less]: @wafakd  We need the` width: 100%;` for the newsComposer! I think that we add a width auto to `#newsActivityEditComposer`. WDYT?

Comment (wafakd) [reply webapp/src/main/webapp/css/news.less]: oui c mieux

**Pull request merged**